### PR TITLE
donation image to support multiple languages

### LINKED ISSUE #316: Replace post donation image to support multiple languages
Currently the picture available for download is not fully translated. Instead we can use the non branded Image without any text.

The text on the image "We're on the road to a Trillion Trees" can be translated and printed on the image as it is generated.

Screen Shot 2020-10-12 at 11 40 46 PM

### PR DESCRIPTION
**Pull request by captainamiedi**
Fixes #316 

Changes in this pull request:
- donate image text to support multiple languages
-
-

@sagararyal

### CODE REVIEW

File: src/features/public/Donations/styles/ThankYou.module.scss [hunk 1]
```diff
@@ -60,7 +60,7 @@
   justify-content: center;
 }
 .thankyouImage {
-  background-image: url('/tenants/planet/images/donations/donationComplete.jpg');
+  background-image: url('/tenants/planet/images/donations/donationComplete_1.jpg');
```
Comment (sagararyal) [inline src/features/public/Donations/styles/ThankYou.module.scss:63]: Please replace the existing image. [Cleanup]

File: src/features/public/Donations/components/ThankYou.tsx [hunk 2]
```diff
@@ -113,6 +113,10 @@ function ThankYou({
 
       <div className={styles.thankyouImageContainer}>
         <div className={styles.thankyouImage}>
+          <div className={styles.thankyouImageHeader}>
+            <p>{t('donate:thankyouHeaderText')}</p>
```
Comment (sagararyal) [inline src/features/public/Donations/components/ThankYou.tsx:117]: If possible please make 1 string use `\n`

File: public/static/locales/en/donate.json [hunk 3]
```diff
@@ -58,5 +58,7 @@
   "noProjectsFound": "No projects found",
   "inTimeOfTaxReturns": "in time for tax returns.",
   "titleToShare": "Planting trees against the climate crisis!",
-  "textToShare": "Preventing the climate crisis requires drastically reducing carbon emissions and planting trees. That’s why I just planted some. Check out {{linkToShare}} if you want to plant some too! #StopTalkingStartPlanting #TrillionTrees @PftP_int"
+  "textToShare": "Preventing the climate crisis requires drastically reducing carbon emissions and planting trees. That’s why I just planted some. Check out {{linkToShare}} if you want to plant some too! #StopTalkingStartPlanting #TrillionTrees @PftP_int",
+  "thankyouHeaderText": "we're on the road on",
```
Comment (sagararyal) [inline public/static/locales/en/donate.json]: 1 string for the Header text.

**Review by sagararyal — CHANGES_REQUESTED** [review]

**Comment by sagararyal** [comment]
Screen Shot 2020-10-17 at 1 20 53 AM

Capitalization;

Also the component for building the image and display are separate; one part is not done in this PR>

File: src/features/public/Donations/components/ThankYou.tsx (hunk 2 above)
Comment (captainamiedi) [reply src/features/public/Donations/components/ThankYou.tsx:117]: Yes we can do it that way but the **Trillion Trees** has a different style if we make it one one sentence, i'm not sure if it is possible to add the styles to it

**Comment by captainamiedi** [comment]
donate tree

These will be the result if i should make it one sentence. it these is ok, i will update the PR

File: src/features/public/Donations/styles/ThankYou.module.scss (hunk 1 above)
Comment (captainamiedi) [reply src/features/public/Donations/styles/ThankYou.module.scss:63]: Done

**Comment by harshvitra** [comment]
Screenshot 2020-10-21 at 1 50 09 AM

we need German translation here for this one line 
@norbertschuler can you give inputs

**Review by harshvitra — APPROVED** [review]

**Review by harshvitra — CHANGES_REQUESTED** [review]
@captainamiedi 
@nirali25parekh 

After donation is complete we have this image at 2 places
1. One which directly pops up (This is translated)
2. We generate an image for sharing (This is what we need to do to complete this feature)

**Comment by captainamiedi** [comment]
@harshvitra I think that image is there. if you click on the download option, the image will be downloaded.

File: public/static/locales/de/donate.json [hunk 4]
```diff
@@ -58,5 +58,6 @@
   "noProjectsFound": "Keine Projekte gefunden",
   "inTimeOfTaxReturns": "rechtzeitig für deine Steuererklärung.",
   "titleToShare": "Bäume pflanzen gegen die Klimakrise!",
-  "textToShare": "Um die Klimakrise zu verhindern, müssen die Kohlenstoffemissionen drastisch reduziert und Bäume gepflanzt werden. Deshalb habe ich gerade einige gepflanzt. Schau unter {{linkToShare}} nach, wenn du auch welche pflanzen willst! #StopTalkingStartPlanting #TrillionTrees @PftP_int"
+  "textToShare": "Um die Klimakrise zu verhindern, müssen die Kohlenstoffemissionen drastisch reduziert und Bäume gepflanzt werden. Deshalb habe ich gerade einige gepflanzt. Schau unter {{linkToShare}} nach, wenn du auch welche pflanzen willst! #StopTalkingStartPlanting #TrillionTrees @PftP_int",
+  "thankyouHeaderText": "Wir sind auf dem Weg zu einer \n <h1>Billion Bäume</h1>"
```
Comment (norbertschuler) [inline public/static/locales/de/donate.json:62]: Text is fine. Will be reviewed and retranslated in LingoHub anyway!

Comment (harshvitra) [reply public/static/locales/de/donate.json:62]: Yes it was for testing purpose

**Review by harshvitra — APPROVED** [review]

**Comment by harshvitra** [comment]
> Screen Shot 2020-10-17 at 1 20 53 AM
> 
> Capitalization;
> 
> Also the component for building the image and display are separate; one part is not done in this PR>

This is Resolved

**Pull request merged**